charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -87,7 +87,7 @@ spec:
             {{- if .Values.controller.sdkDebugLog }}
             - --aws-sdk-debug-log=true
             {{- end}}
-            - --logtostderr
```

Comment:
What will happen if users supply this CLI option in the new release?

Reply:
This CLI option was previously added by [klog.InitFlags](https://github.com/kubernetes/klog/blob/2ca9ad30301bf30a8a6e0fa2110db6b8df699a91/klog.go#L415) which is replaced by [klog.AddFlags](https://github.com/kubernetes/component-base/blob/81045940b8278d6af9607669bec8aa748f3965ab/logs/api/v1/options.go#L185) in this PR and does _not_ include this flag, see [Removed klog flags](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components#removed-klog-flags). 

Newest revision adds an equivalent flag (disabled by default) that we should plan to remove in a future release.